shell/common/shell.h

```diff
@@ -169,6 +183,27 @@ class Shell final : public PlatformView::Delegate,
       CreateCallback<PlatformView> on_create_platform_view,
       CreateCallback<Rasterizer> on_create_rasterizer);
 
+  //----------------------------------------------------------------------------
+  /// @brief      Creates a shell instance using the provided settings.
+  /// @details    This version of Create can take in a running DartVMRef and a
+  ///             function that defines how the Shell's Engine should be
+  ///             created.
+  /// @param[in]  vm             A running DartVMRef where this Shell's Dart
```

Comment:
Do you want to briefly describe how one typically gets a DartVMRef?

Reply:
I don't know if we should point to DartVMRef::Create, it's pretty straightforward